### ISSUE TITLE
Add image gallery which previews the themes

### ISSUE THREAD

**Issue by Siedlerchr** [JabRef/themes.jabref.org#10]
[It](url) would be nice to have a preview of the theme in this repository!

**Comment by Siedlerchr** [comment]
possible idea https://github.com/opieters/jekyll-image-gallery-example/tree/gh-pages
https://dmnfarrell.github.io/software/jekyll-galleries

**Comment by ghost** [comment]
I was thinking about this idea too!
I'm not a programmer so I hope somebody can make this.
I keep posting the screenshots here.

**Comment by DiamondMyK** [comment]
Hi, I'm new to open source and want to try to solve some simple problems. Can you assign this issue to me? I would appreciate it!

**Comment by jeanprbt** [comment]
Hi, I am a university student in Software Engineering Fundamentals at KTH, Stockholm. We, as a group of 5 students, would like to work on this issue. Could you assign us ? (@JEANPRBT, @rickardo-cornelli, @lola20b, @zaina-ram, @Raahitya-14)

**Comment by Siedlerchr** [comment]
I can only assign one person, but you are welcome to work on a solution!

**Comment by jeanprbt** [comment]
I am so sorry, I had not noticed this was on the secondary project themes.jabref.org.
I will find another issue since we are required to work on the main project. You can un-assign me, I apologize for the inconvenience.

**Comment by Siedlerchr** [comment]
Ah okay, no problem

**Comment by Siedlerchr** [comment]
@JEANPRBT  Mabye you can work on https://github.com/JabRef/jabref/issues/10661 there hasn't been any activity since a while

**Comment by jeanprbt** [comment]
Yes I am interested ! You can assign me on this one (https://github.com/JabRef/jabref/issues/10661).

**Comment by jiaxin0103** [comment]
Hello, I am a student at the Australian National University. For our team project, we need to contribute to an open-source project. May I be assigned to this task?

**Comment by NEBANU** [comment]
I am a member of the Australian National University team.
The issue has been solved on the main repo: https://github.com/JabRef/jabref/pull/12104
